x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiBasicTable, EuiBasicTableColumn, EuiButtonIcon, EuiCopy } from '@elastic/eui';
+import { CriteriaWithPagination } from '@elastic/eui/src/components/basic_table/basic_table';
+import { useActions, useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+
+import { CredentialsLogic } from '../credentials_logic';
+import { IApiToken } from '../types';
+import { TOKEN_TYPE_DISPLAY_NAMES } from '../constants';
+import { apiTokenSort } from '../utils/api_token_sort';
+import { getModeDisplayText, getEnginesDisplayText } from '../utils';
+
+export const CredentialsList: React.FC = () => {
+  const { deleteApiKey, fetchCredentials, showCredentialsForm } = useActions(CredentialsLogic);
+
+  const { apiTokens, meta } = useValues(CredentialsLogic);
+
+  const items = useMemo(() => apiTokens.slice().sort(apiTokenSort), [apiTokens]);
+
+  const columns: Array<EuiBasicTableColumn<IApiToken>> = [
+    {
+      name: 'Name',
+      width: '12%',
+      render: (item: IApiToken) => item.name,
```

Comment:
Oh, something I totally meant to mention yesterday and forgot - are we cool with `item` as a fairly generic arg or do we want to consider at least changing it to something a bit more specific to this file/table? e.g.,

```suggestion
      render: (token: IApiToken) => token.name,
```

I think it helps with context a little (e.g. `token.key`, `token.type`, etc.)!

Reply:
Switched them all over: https://github.com/elastic/kibana/pull/79749/commits/a0ff6461eb37e6e3b2418785a5e73c1a8eefcc63?w=1